test/index.test.js

```diff
@@ -222,6 +223,42 @@ describe('rendering', function () {
     expect(counterSegments.at(2).props().digitMap).toEqual(digitMap)
     expect(counterSegments.at(3).props().digitMap).toEqual(digitMap)
   })
+
+  it('passes correct labels to segments', function () {
```

Comment:
why do map and function covered by one test? Better to make different tests for them